evap/evaluation/templates/confirmation_text_modal.html

```diff
@@ -0,0 +1,46 @@
+{% load evaluation_filters %}
+
+<div class="modal fade" id="{{ modal_id }}" tabindex="-1" role="dialog" aria-labelledby="{{ modal_id }}Label" aria-hidden="true">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="{{ modal_id }}Label">{{ title }}</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span class="fas fa-times"></span>
+                </button>
+            </div>
+            <form id="{{ modal_id }}Form">
+                <div class="modal-body">
+                    {{ question|safe }}
+                    <br /><br />
+                    <div class="form-group">
+                        <input type="text" class="form-control" id="{{ modal_id }}ConfirmationText" oninput="{{ modal_id }}CheckValue();" />
+                    </div>
+                </div>
+                <div class="modal-footer">
+                    <button type="button" id="{{ modal_id }}ActionButton" class="btn btn-{{ btn_type }}" data-dismiss="modal" disabled>{{ action_text }}</button>
+                    <button type="button" class="btn btn-light" data-dismiss="modal">{% trans 'Cancel' %}</button>
+                </div>
+            </form>
+        </div>
+    </div>
+</div>
+
+<script type="text/javascript">
+    function {{ modal_id }}Show(dataId, dataLabel) {
+        // call the modal's action function when action button was pressed and give dataId as parameter
+        $('#{{ modal_id }} #{{ modal_id }}ActionButton').unbind().click(function(){ {{ modal_id }}Action(dataId); });
+        // put the dataLabel text in the right place
+        $('#{{ modal_id }} #data-label').text(dataLabel);
+        // show modal
+        $('#{{ modal_id }}').modal();
+    }
+
+    function {{ modal_id }}CheckValue() {
+        if ($('input#{{ modal_id }}ConfirmationText').val() == $('#{{ modal_id }} #data-label').text()) {
```

Comment:
Maybe make this case insensitive? For people actually typing the name that might be easier.

Reply:
it's all about making it not easy :)
i thought it would be better to ask for the exact same name but i can change that if you don't like it.

Reply:
I just thought that forcing the correct capitalization will probably make people copy and paste the name from the page. Over time, this would circumvent this check, but since you would not do this every day, it's probably not that much of an issue.

(Its done differently in popular places anyway - Github security check for deleting a repository forces you to enter its name but allows different capitalization. The apt package manager will make you type "Yes, do as I say!" in that exact capitalization when doing something that will probably break the system)